packages/fxa-settings/src/pages/ResetPasswordWithRecoveryKeyVerified/index.tsx

```diff
@@ -0,0 +1,70 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { RouteComponentProps, navigate } from '@reach/router';
+import { Localized } from '@fluent/react';
+import { logViewEvent } from '../../lib/metrics';
+import { useNavigate } from '@reach/router';
+import Ready from '../../components/Ready';
+
+type ResetPasswordWithRecoveryKeyVerifiedProps = {
+  serviceName?: string;
+};
+
+const ResetPasswordWithRecoveryKeyVerified = ({
+  serviceName,
+}: ResetPasswordWithRecoveryKeyVerifiedProps & RouteComponentProps) => {
+  const navigate = useNavigate();
+  const viewName = 'settings.reset-password-with-recovery-key-verified';
+  const baseActionName = 'reset_password_with_recovery_key_verified';
+
+  // TO-DO: Attach correct ACTUAL actions to both the link and the button
+
+  return (
+    <>
+      <Ready
```

Comment:
Small nit, syntactic sugar should be:

```
<Ready
  {...{ viewName, serviceName, baseActionName }} 
/>
```

Reply:
I think we might be missing some kind of processing for these files (I guess it could just be me?) I've repeatedly gotten errors when I try to use syntactic sugar, specifically thinking that I'm trying to pass in a bool instead of the associated value

Reply:
nope it was just me using the wrong syntax!